vnext/Microsoft.ReactNative/Version.rc

```diff
@@ -10,6 +10,10 @@
 #define VER_FILEVERSION_STR XSTRINGIZE(RNW_VERSION)
 #define VER_FILEVERSION RNW_MAJOR,RNW_MINOR,RNW_PATCH,RNW_PKG_VERSION_BUILD
 
+#define VER_PRODUCTVERSION_STR XSTRINGIZE(RNW_VERSION+RNW_COMMITID)
+#define VER_PRODUCTVERSION RNW_MAJOR,RNW_MINOR,RNW_PATCH,RNW_PKG_VERSION_BUILD
```

Comment:
Should there be commit_id at the end of this field value ?

Reply:
That is not allowed, it is a 4-digit version number. I opted to create the sibbling of `FILEVERSION` for symetry. I could have kept `VER_FILEVERSION` but that felt inconsistent.